source/reference/projects.rst

```diff
@@ -0,0 +1,186 @@
+======================
+ Projects - Reference
+======================
+
+
+.. seealso::
+
+   :doc:`Documentation on projects </file_management/projects>`
+      Explains how to work with projects.
+
+
+Project information is saved in meta files
+that can be edited
+to allow further configuration
+than just adding or removing folders.
+You can edit the project file
+of the currently active non-anonymous project
+via the **Project → Edit Project** menu.
```

Comment:
Add *The anonymous project cannot be configured in any way, because no .sublime-project file exists for it.*.